docs/parameters.md

```diff
@@ -3,25 +3,27 @@ There are several optional parameters that may be passed into `CreateVolumeReque
+| "iopsPerGB"                  |                                                    |         | I/O operations per second per GiB. Can be specified for IO1, IO2, and GP3 volumes.                                                                                                                                                                                                                                                                                                             |
+| "allowAutoIOPSPerGBIncrease" | true, false                                        | false   | When `"true"`, the CSI driver increases IOPS for a volume when `iopsPerGB * <volume size>` is too low to fit into IOPS range supported by AWS. This allows dynamic provisioning to always succeed, even when user specifies too small PVC capacity or `iopsPerGB` value. On the other hand, it may introduce additional costs, as such volumes have higher IOPS than requested in `iopsPerGB`. |
+| "iops"                       |                                                    |         | I/O operations per second. Can be specified for IO1, IO2, and GP3 volumes.                                                                                                                                                                                                                                                                                                                     |
+| "throughput"                 |                                                    | 125     | Throughput in MiB/s. Only effective when gp3 volume type is specified. If empty, it will set to 125MiB/s as documented [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html).                                                                                                                                                                                      |
+| "encrypted"                  | true, false                                        | false   | Whether the volume should be encrypted or not. Valid values are "true" or "false".                                                                                                                                                                                                                                                                                                             |
+| "blockExpress"               | true, false                                        | false   | Enables the creation of [io2 Block Express volumes](https://aws.amazon.com/ebs/provisioned-iops/#Introducing_io2_Block_Express) by increasing the limit for io2 volumes to 256000 for io volumes.                                                                                                                                                                                              |
```

Comment:
nit: "by increasing the limit for io2 volumes to 256000 for io volumes." -> "by increasing the IOPS limit for io2 volumes to 256,000." Would be good to also add a note here specifying this parameter is only supported for certain instance types.

Reply:
Reworded, also added a note about instance types.